src/hdmf/backends/hdf5/h5tools.py

```diff
@@ -280,10 +280,7 @@ def write(self, **kwargs):
             for ns_name in ns_catalog.namespaces:
                 ns_builder = self.__convert_namespace(ns_catalog, ns_name)
                 namespace = ns_catalog.get_namespace(ns_name)
-                if namespace.version is None:
-                    group_name = '%s/unversioned' % ns_name
-                else:
-                    group_name = '%s/%s' % (ns_name, namespace.version)
+                group_name = '%s/%s' % (ns_name, namespace.version)
```

Comment:
I changed the default to be `None`. This would make the group name "None". I don't have a problem with that, since I don't think it would impact reading the namespace

Reply:
cool.  When it is loaded from a file, would `"None"` string be casted to a proper `None` for the `.version`? Just checking, didn't spot anything obvious in the diff

Reply:
This is now resolved in `SpecNamespace.__init__`

Reply:
@yarikoptic Thanks for checking that!